recipes/galaxy-containers/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "19.9.0" %}
+{% set sha256 = "8951ce2fa0532604a9383ee92b711cd41d026f2d50e2a29d236fb4295a51b218" %}
+
+package:
+  name: galaxy-containers
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/g/galaxy-containers/galaxy-containers-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    # Depend on galaxy-util during build time to avoid packaging galaxy/__init__.py .
+    - galaxy-util
+  run:
+    - galaxy-util
```

Comment:
I'd consider this a library mostly, so my gut feeling is no, the environment should be restricted in whatever app you build from it.

Reply:
I was thinking in terms of whether we should tie the versions of `galaxy-containers` (or `galaxy-tool-util` in the other recipe) to those from `galaxy-util`. This would be to avoid that users get `galaxy-util=19.9` installed for whatever reason but also `galaxy-containers>=20` which might requires some functionality from `galaxy-util>=20`.
Meaning, something like
```yaml
requirements:
host:
    - galaxy-util >={{ version }}
  run:
    - galaxy-util >={{ version }}
```

Reply:
Sounds good 👍

Reply:
Does `galaxy-containers >={{ version }}` for the `galaxy-tool-util` recipe also make sense?

Reply:
I'd say yes, I think we can do this uniformly to all packages